app/iot/service.py

```diff
@@ -39,11 +39,11 @@ def unregister_device(self, device_id: str) -> None:
     def get_device(self, device_id: str) -> Device:
         return self.devices[device_id]
 
-    def run_program(self, program: list[Message]) -> None:
+    async def run_program(self, program: list[Message]) -> None:
         print("=====RUNNING PROGRAM======")
         for msg in program:
-            self.send_msg(msg)
+            await self.send_msg(msg)
```

Comment:
From the checklist: After that, gather running programs (in service.py) - here the code also may be easy. And if you run it — wow, it works so fast, asyncio is powerful! But look at the logic — we've broken it: we may start playing music on Speaker before it was turned on, same with flushing and cleaning Smart Toilet. So, despite it working fast, it works incorrectly.